tests/e2e/pages/AnalyticsOverview.ts

```diff
@@ -0,0 +1,48 @@
+import { waitForElementByText } from '../utils/actions';
+import { Analytics } from './Analytics';
+
+export class AnalyticsOverview extends Analytics {
+	async navigateTo() {
```

Comment:
Is this being used for anything or planned for future use?

Reply:
That was suppose to overwrite the main `navigate` button, instead of `navigateTo` and use it in the test. I will have to change that, thanks for pointing that out.